types/jasmine/index.d.ts

```diff
@@ -197,8 +197,9 @@ declare function spyOnProperty<T>(object: T, property: keyof T, accessType?: "ge
 /**
  * Installs spies on all writable and configurable properties of an object.
  * @param object The object upon which to install the `Spy`s.
+ * @param includeNonEnumerable Whether or not to add spies to non-enumerable properties.
  */
-declare function spyOnAllFunctions<T>(object: T): jasmine.SpyObj<T>;
+declare function spyOnAllFunctions<T>(object: T, includeNonEnumerable?: boolean): jasmine.SpyObj<T>;
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54255 covers this, so just be sure that your rebase handles this right.